sections/06-vibrant-bodies.md

```diff
@@ -0,0 +1,201 @@
+
+Mulgan, Steinberg, and Salem note that open source methods are "at the cusp of familiar tools," often resembling conversations, formal research teams, academic scholarship, and DIY communities (15-6). Amongst the characteristics of open source methodologies described in _Wide Open_, several are particularly germane to my practice: transparency, peer review and feedback loops, and incrementalism (16). These strategies have helped to expand the networks of use and criticism for my work beyond my own local scholarly and creative communities. Work on _Flocking_, in particular, has been substantially supported by the very diverse perspectives that make up its community of users, contributors, and critics. By working transparently so that each change to the system (known as a "commit") is visible to those following the project using the Github collaboration site<sup>4</sup>, Flocking's community is able to offer feedback, advice, and critical improvements while I work.
+
+As an artist, however, working transparently and offering up a work-in-progress openly is a strange prospect, particularly when the project is still in the early stages of conception, before it has formed from a tangle of curious intensities into an object for use, discussion, or reception. Yet, as a result of my own openness and willingness to risk visibility in the tentative stages of creation, members of the community have been motivated to share their artistic projects reciprocally. I have benefitted from this mutual exchange significantly, not only gaining new design insights into how to support the needs and processes of artists who are significantly unlike me<sup>5</sup>, but also learning new creative strategies by studying, experiencing, and working through the software art created by others in the community. For example, the approach I used while working with recorded sound in _Font Màgica_ was substantially informed by my study of granular synthesis instruments created and shared by Alex Geddie, whose music for _Constructed Land_ subtly lenses recordings of "classical" repertoire into new works that occupy a strange, buzzing, stuttering middle ground between quotation and transformation<sup>6</sup>. In the past, as an instrumental music composer, I was rarely able to have such ready access to the scores and recordings of "non-master" composers except those within my immediate (though nonetheless highly germane) sphere of artistic friendships within Toronto. To be clear, this is not an argument for global networks and against local ones; rather, it is a acknowledgement of the new forms of locality that can emerge "telesthetically" around the creative currents and capabilities of shared, networked artifacts and practices.
```

Comment:
This may be inherent in `for use, discussion or reception`, but perhaps it's worth being so explicit as to add "extension, appropriation, adaptation" or similar. Perhaps consider this also in terms of the language used in early open source projects of "use at your own risk, not production ready, etc" that attempts to dissuade others from use while still being in the open.